contrib/legal/tos.html.sample

```diff
@@ -130,7 +130,7 @@
 
   <p>If you'd like to use our trademarks, you must follow all of our trademark guidelines.</p>
 
-  <p>This Agreement is licensed under <a href="https://creativecommons.org/publicdomain/zero/1.0/">CCO Public Domain License</a>.</p>
+  <p>This Agreement is licensed under CC0 Public Domain License. See <a href="https://creativecommons.org/publicdomain/zero/1.0/legalcode">full legal code here</a>.</p>
```

Comment:
```suggestion
  <p>This Agreement is licensed under <a href="https://creativecommons.org/publicdomain/zero/1.0/legalcode">CCO Public Domain License</a>.</p>
```
For accessibility purposes this structure is recommended per WCAG 2.0 so that it is easier for screenreaders to parse.